### PR TITLE
Adds filter labels for piece manager

### PR DESCRIPTION
**Pull request by abea**
Resolves https://apostrophecms.atlassian.net/browse/A30U-295

### CODE REVIEW

**Review by falkodev — APPROVED** [review]

**Comment by falkodev** [comment]
I don't understand what was wrong or missing before. But no harm in this PR.

File: modules/@apostrophecms/ui/ui/apos/components/AposFilterMenu.vue [hunk 1]
```diff
@@ -65,9 +66,7 @@ export default {
           field: {
             name: filter.name,
             type: 'radio',
-            // TODO: Remove `|| filter.name.toUpperCase()` once filters include
```
Comment (boutell) [inline modules/@apostrophecms/ui/ui/apos/components/AposFilterMenu.vue:68]: Should we support a more graceful automatic naming if label is missing? We do for schema fields. It's a bit of a parlor trick, but devs do appreciate that "save me all the time" stuff.

```
        if ((!field.label) && (!field.contextual)) {
          field.label = _.startCase(field.name.replace(/^_/, ''));
        }
```

Does a reasonably pretty job.

Comment (abea) [reply modules/@apostrophecms/ui/ui/apos/components/AposFilterMenu.vue:68]: At the moment it fails to have the filter label missing, but doesn't break the app. I'd be alright with defaulting to the `name` values. I'm hesitant to install a utility for this kind of thing. I'd rather make it clear that something needs fixing and how to fix it.

**Comment by abea** [comment]
@falkodev The filter labels were displaying incorrectly because we didn't have `label` values for them and so they would end up displaying all-caps rather than title-case as in the designs.

File: modules/@apostrophecms/ui/ui/apos/components/AposFilterMenu.vue (hunk 1 above)
Comment (boutell) [reply modules/@apostrophecms/ui/ui/apos/components/AposFilterMenu.vue:68]: Yeah, if we do it, it should be done server side where lodash is already available, and the configuration that reaches Vue should be ready to rock with a label. Which, puts it outside the scope of your PR. Approving.

**Pull request merged**